_docs-sources/intro/overview/josh-feedback.md

```diff
@@ -0,0 +1,66 @@
+---
+'title': "Josh's Feedback"
+---
+
+# Josh's Feedback
+
+- What's the vision for "tool fundamentals"?
+  - How does it compare to Set Up Your Environment / Installing Neccessary Tools?
+  - Should "Setup Your Tools" (formerly Tool Fundamentals) be part of Set Up Your Environment?
+- "Next Steps" at bottom should be the same boldness. 
+- Should "Deploy Your First Module" focus on teaching The Gruntwork Way? 
+  - You could start by identifying a use case ("deploy a VPC"), then look at the service catalog, then, use Terraform or Terragrunt to deploy the module (we should cover both).
+- Zooming out leads to centered text. Should we left-align instead?
+- Are inline links too low-contrast (compared to black text) to easily see?
+
+# Bugs
+
+## Spacing on multi-level outlines
+
+### This works fine
+
+- What is Gruntwork
+  - DevOps accelerator
+
+### But this does not
+
+- What is Gruntwork
+  - DevOps accelerator
+    - Pre-written, production-grade AWS infrastructure-as-code lets you go live in days
+      - with all of the pieces you need to go to production including:
+        - **Landing Zones**
+        - **Docker Orchestration:** Kubernetes, ECS, Nomad
+        - **CI/CD Pipelines:** Jenkins, CircleCI, TravisCI
+
+- Products
+  - Service Catalog 
+
+### Another bug example
+
+- **DevOps Skill.** Gruntwork requires that users learn the following core DevOps skills:
+  - Terraform: Intermediate
+  - Bash: Beginner
+  - Packer: Beginner
+  - Docker: Intermediate
+    :::note
+
+    In the Gruntwork model, the user takes responsibility for developing these skills, however Gruntwork assists this effort by offering [DevOps Courses](/courses), [Professional Services](#), and [Support](#).
+
+    :::
+- **Ownership.** sdfadfasdfdfdf
```

Comment:
This one is less obvious, but it appears to me that the particular Markdown renderer that Docusaurus is using treats all the bullets as paragraphs if any of them include a block-level element. That's just my assumption, though.

That said, I find this result reasonable under the circumstances. Otherwise, the admonition would stand apart more from the bullet it relates to than the other bullets would from each other. I think it's just a strange use case to put an admonition inside a list, especially when the contents of the bullets _aren't_ otherwise long form.